### PR TITLE
KOGITO-1412: Added refresh button on Domain explorer

### PR DESCRIPTION
**Pull request by Sara4994**

### CODE REVIEW

File: packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx [hunk 1]
```diff
@@ -247,6 +253,13 @@ const DomainExplorerColumnPicker: React.FC<IOwnProps> = ({
 
     return unique;
   }
+
+  const onRefresh = () => {
+    if(enableRefresh && parameters.length > 1) {
+    // tslint:disable-next-line: no-floating-promises
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx]: can we avoid disabling the tslint rule?

**Review by cristianonicolai — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]
looks good ! :+1:

File: packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx (hunk 1 above)
Comment (Sara4994) [reply packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx]: @cristianonicolai done

File: packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx [hunk 2]
```diff
@@ -363,6 +370,9 @@ const DomainExplorerColumnPicker: React.FC<IOwnProps> = ({
           <Button variant="primary" onClick={generateQuery}>
             Apply columns
           </Button>
+          <Button variant="plain" onClick={onRefresh} className="pf-u-m-md">
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx]: We need an aria-label for screenreaders. You can use this:
```
          <Button variant="plain" onClick={onRefresh} className="pf-u-m-md" aria-label={"Refresh list"}>
```

Comment (Sara4994) [reply packages/management-console/src/components/Organisms/DomainExplorerColumnPicker/DomainExplorerColumnPicker.tsx]: @srambach  done

File: packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.css [hunk 3]
```diff
@@ -6,4 +6,8 @@
     --pf-c-table--nested--first-last-child--PaddingLeft: 0;
     margin-bottom: var(--pf-global--spacer--lg);
     max-width: 80vw;
+}
+
+.kogito-management-console--domain-explorer__table .pf-c-table__toggle{
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.css:11]: I'm not sure why this is here? We should always use the spacer variables, but I think the spacing was fine here without any adjustment.

Comment (cristianonicolai) [reply packages/management-console/src/components/Organisms/DomainExplorerTable/DomainExplorerTable.css:11]: @srambach I think the problem is that we don't add any content when value is null, we can review that if we need another placeholder to be displayed which would avoid this fix.

**Pull request merged**